internal/config/gql_provider.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"sigs.k8s.io/yaml"
+)
+
+// GqlProvider is GraphQL provider
+type GqlProvider struct {
+	gqlClient *GqlClient
+}
+
+// NewGqlProvider initializes new GraphQL config source provider
+func NewGqlProvider(gql *GqlClient) *GqlProvider {
+	return &GqlProvider{gqlClient: gql}
+}
+
+// Configs returns list of config files
+func (g *GqlProvider) Configs(ctx context.Context) ([]string, error) {
+	d := os.Getenv("CONFIG_SOURCE_IDENTIFIER")
+	if d == "" {
+		return nil, nil
```

Comment:
Maybe we could return here an error? WDYT? 🤔

Reply:
This part is also outdated, please check new content